user_guide/src/examples/dataframe/date_groupby.py

```diff
@@ -0,0 +1,9 @@
+import polars as pl
+from .datatypes import df
+
+
+res = (
+    df.groupby([pl.col("date").dt.year().alias("year"), pl.col("date").dt.month().alias("month"), pl.col("color")])
+    .agg([pl.col("rank").alias("best_of_month").min()])
```

Comment:
trailing comma's make this formatting much better.

Reply:
Marking these as resolved, but they're more like *potentially resolved*. Lmk if I'm not doing it properly.